src/utils/ledger.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { IPC_RENDERER } from '../constants';
+import hathorLib from '@hathor/wallet-lib';
+
+/**
+ * Error thrown when we get an error from Ledger
+ *
+ * @memberof Ledger
+ * @inner
+ */
+export class LedgerError extends Error {}
+
+let ledger = {};
+
+if (IPC_RENDERER) {
+  /**
+   * Methods to interact with ledger
+   *
+   * @namespace Ledger
+   */
+  ledger = {
```

Comment:
Why don't you create a class to handle it? It seems easier to use and maintain.

Reply:
There's no state to keep, so I don't see any advantage in having a class. We could have a class with only `classmethod`, but I think it's about the same thing.

Reply:
Ok. Agreed.